### PR TITLE
Travis 

### PR DESCRIPTION
**Pull request by dukex**
A initial script to validate the alphabetic order, the idea was mentioned on #342

### CODE REVIEW

**Comment by dukex** [comment]
ping @mholt @shurcooL @avelino

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - 1.3
```
Comment (shurcooL) [inline .travis.yml]: Let's use the current version of Go, 1.4.

Comment (dukex) [inline .travis.yml]: Done!

**Comment by shurcooL** [comment]
Nice!

I think we should give it a spin.

File: repo_test.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+func testList(t *testing.T, list *goquery.Selection) {
+	list.Find("ul").Each(func(_ int, items *goquery.Selection) {
+		testList(t, items)
+		items.RemoveFiltered("ul")
+	})
+	checkAlphabeticOrder(t, list)
+}
+
+func readme() []byte {
+	input, _ := ioutil.ReadFile("./README.md")
```
Comment (shurcooL) [inline repo_test.go]: Instead of ignoring error, you could panic if it's not nil.

File: repo_test.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	html := append([]byte("<body>"), blackfriday.MarkdownCommon(input)...)
+	html = append(html, []byte("</body>")...)
+	return html
+}
+
+func startQuery() *goquery.Document {
+	ioutil.WriteFile("z.html", readme(), os.ModePerm)
```
Comment (shurcooL) [inline repo_test.go]: Is this needed? `z.html` doesn't seem to be used anywhere.

**Comment by dukex** [comment]
I create a test pull request in my fork https://github.com/dukex/awesome-go/pull/1 
The result https://travis-ci.org/dukex/awesome-go

File: repo_test.go (hunk 3 above)
Comment (dukex) [inline repo_test.go]: oh, I was using to debug

**Comment by avelino** [comment]
Very very good man!

**Comment by mholt** [comment]
Woah, that's awesome. I don't have a chance to test it out right now but I liked what I saw on your fork! I approve.

**Comment by avelino** [comment]
We can use the same process to generate the site in HTML, so the Google indexes the content generated! Not just javascript.

I will accept the PR, OK :D

**Comment by shurcooL** [comment]
Looks like the current README.md has some alphabetization issues that are being caught. We can fix those as part of this PR, so that it results in a passing Travis build.

You can also add a Travis build badge to the README, for an example see the change to REAMDE.md in https://github.com/shurcooL/gostatus/commit/9969324c2a427d9403bdea5ca6541d0dc9d7fe50.

**Comment by dukex** [comment]
All Green https://travis-ci.org/dukex/awesome-go/builds/58388932

**Comment by shurcooL** [comment]
Very nice! LGTM.

**Pull request merged**